soroban-env-common/env.json

```diff
@@ -711,7 +711,39 @@
                     ],
                     "return": "u64",
                     "docs": "Convert a `Duration` object a `u64`."
-                }                
+                },
+                {
```

Comment:
Could you please move these near the respective u256/i256 function declarations? We could even change the exports given that this is going to be a 'breaking' release anyway.